src/shared/components/ActionBar/ActionBar.style.ts

```diff
@@ -78,7 +78,12 @@ export const DetailsContainer = styled.div`
   margin-left: auto;
   display: flex;
   align-items: center;
-  padding: 0 ${sizes(5)};
+  padding: 13px ${sizes(4)};
```

Comment:
I think we actually don't need any vertical padding as it should fill the entire available height anyway, right? For horizontal padding, design uses 20px